css/impact.css

```diff
@@ -0,0 +1,286 @@
+}
+
+/* icon section code */
+.yellow-background {
+	background: var(--color-yellowgreen);
+	background-size: auto 95%;
+	background-repeat: no-repeat;
+	background-position: center center;
+}
+.icons-parent {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+	background-color: var(--color-onyx);
+	border-radius: var(--spacing10);
+	padding-top: var(--spacing80);
+	padding-bottom: var(--spacing80);
+	margin: 0 auto;
+	gap: var(--spacing40);
+}
+
+.icon-child {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-content: center;
+	text-align: center;
+}
+
+.icon-child img {
+	margin: 0 auto;
+}
+.icon-child .h5-styles {
+	color: var(--color-white);
+	margin: var(--spacing40) 0 0 0;
+}
+
+/* 
+===================================
+Media Queries (overriding styles for different screen sizes)
+
+Note: Here we are using a mobile-first approach, so we start with the smallest screen size and work our way up.
+Start by writing styles for the mobile version of the design on the lines above.
+Anything that needs to change for larger screens should be added to the media queries below.
+
+Feel free to change the min/max-width values to whatever you need. Use variables.
+===================================
+*/
+
+/* Tablet Only */
+@media screen and (min-width: 768px) and (max-width: 1024px) {
+	/* Add styles that only apply to tablet. These are usually fixes to get resposnivenes right */
+}
+
+/* Tablet and Larger */
+@media screen and (min-width: 768px) {
+	/* Add styles that apply to table and desktop */
+	/* Title/Hero section */
+	.title-img {
+		display: none;
+	}
+	/* adding desktop image */
+	.hero {
+		height: 100vh;
+		max-height: 1000px;
+		background-image: url("/assets/images/pages/our-impact/desktop/vr-ar-d.jpg");
+		background-size: cover;
+		background-position: center;
+		background-repeat: no-repeat;
+	}
+	.title-text {
+		width: 100%;
+		height: 100%;
+		background: linear-gradient(
+			92deg,
+			rgba(9, 9, 9, 0.77) 40.35%,
+			rgba(9, 9, 9, 0.74) 44.51%,
+			rgba(9, 9, 9, 0.71) 48.63%,
+			rgba(9, 9, 9, 0) 83.13%
+		);
+
+		/* aligning the title text div */
+		display: flex;
+		align-items: center;
+	}
+	.title-text div {
+		/* adding width instead of margin for responsiveness */
+		width: 70%;
+		margin: 0 auto;
+
+		display: flex;
+		flex-direction: column;
+	}
+	.title-text p,
+	.title-text .h2-styles {
+		/* adding width to make sure text stays on left side */
+		width: 40%;
+		text-align: left;
+		color: var(--color-white);
+	}
+	.title-text .h2-styles {
+		font-size: var(--font56);
+		line-height: var(--line-height58);
+	}
+
+	
+
+	/* icon section */
+	.icons-parent {
+		flex-direction: row;
+		flex-wrap: wrap;
+		justify-content: center;
+	}
+	.icon-child {
+		min-width: 40%;
+	}
+	.yellow-background {
+		background-size: auto 90%;
+	}
+}
+
+/* Desktop Only */
+@media screen and (min-width: 1024px) {
+	/* Add styles that apply specifically to desktop */
+  
+	/* top and bottom section */
+
```

Comment:
I think the title needs to be adjusted in alignment and spacing!
![image](https://github.com/indigenousfriends/itlct-website-redesign/assets/98284615/b61d334e-72cf-4901-aba7-e8db24f89c59)

Please let me know if there is anything i can clarify!

Reply:
taken care of by Charli